pkg/rpc/methods.go

```diff
@@ -281,6 +278,17 @@ type L2SyncProgress struct {
 	HighestBlockID *big.Int
 }
 
+// isSyncing returns true if the L2 execution engine is syncing with L1.
+func (p *L2SyncProgress) isSyncing() bool {
+	if p.SyncProgress != nil ||
+		p.CurrentBlockID == nil ||
+		p.HighestBlockID == nil ||
+		p.CurrentBlockID.Cmp(p.HighestBlockID) < 0 {
+		return true
+	}
+	return false
```

Comment:
Why not simply `return p.SyncProgress != nil ||
		p.CurrentBlockID == nil ||
		p.HighestBlockID == nil ||
		p.CurrentBlockID.Cmp(p.HighestBlockID) < 0` ?

Reply:
Refactoring to make this line of code more readable.

Reply:
fixed. But I don't think it's a good idea to have such a long judgment statement, the function should judge the false case first and return true at the end.